content/en/service_management/service_level_objectives/monitor.md

```diff
@@ -119,6 +119,8 @@ If you pause a Synthetic test, the SLO removes the time period with missing data
 
 ## Underlying monitor and SLO histories
 
+<div class="alert alert-warning">Support for SLO Replay will be deprecated in the near future. Impacted customers will be notified about the change ahead of time with details about how they will be impacted.</div>
```

Comment:
Can we provide a more specific timeline for deprecation, even if it's just a month or a quarter? Thank you!

Reply:
@jhgilbert can we say next few months?